website/mails.py

```diff
@@ -108,7 +108,7 @@ def send_mail(to_addr, mail, mimetype='plain', from_addr=None, mailer=None,
         mail_server=mail_server
     )
 
-    if settings.USE_CELERY:
+    if settings.USE_CELERY and settings.USE_EMAIL:
```

Comment:
The check for `USE_EMAIL` should be outside this block so that it's checked even if USE_CELERY is False